views/index.pug

```diff
@@ -0,0 +1,26 @@
+extends layout
+
+block content
+  h1 Number Guess Game
```

Comment:
Just a sanity check here. This view actually works? Maybe the indentation in github is off, but I would double check indentation on this file. Not a major issue if this works. Just readability.